content/blog/2017/2017-01-18-refactoring-conditional-to-pipeline.adoc

```diff
@@ -0,0 +1,475 @@
+---
+layout: post
+title: "Refactoring Conditional BuildSteps into Jenkins Pipeline"
```

Comment:
Not sure I'd call it "refactoring", as this is akin to a platform change. "Converting" perhaps?